# Define the goal
uint32 method # Specify which method we want to use
string[] object_names # which objects do you want to grasp
---
# Define the result
geometry_msgs/PoseStamped[] grasp_poses
vision_msgs/BoundingBox3DArray object_bbs
---
# Define a feedback message
float32 percent_complete